src/xcode/ENA/ENA/Source/Services/CoronaTestService/CoronaTestService.swift

```diff
@@ -397,30 +397,36 @@ class CoronaTestService {
 			completion(.failure(.noRegistrationToken))
 			return
 		}
+			let resource = RegistrationTokenResource(
+				sendModel: SendRegistrationTokenModel(
+					token: registrationToken
+				)
+			)
+			restServiceProvider.load(resource) { result in
+				switch result {
+				case .success(let model):
+					let submissionTAN = model.submissionTAN
+					switch coronaTestType {
+					case .pcr:
+						self.pcrTest?.submissionTAN = submissionTAN
+						self.pcrTest?.registrationToken = nil
 
-		client.getTANForExposureSubmit(forDevice: registrationToken, isFake: false) { result in
-			switch result {
-			case let .failure(error):
-				Log.error("[CoronaTestService] Getting submission tan failed: \(error.localizedDescription)", log: .api)
+						Log.info("[CoronaTestService] Received submission tan for PCR test: \(private: String(describing: self.pcrTest), public: "PCR Test result")", log: .api)
```

Comment:
```suggestion
						Log.info("Received submission tan for PCR test: \(private: String(describing: self.pcrTest), public: "PCR Test result")", log: .api)
```
no longer needed

Reply:
shouldn't we keep the  "[CoronaTestService]" to make it easier to debug the logs?

Reply:
The class, method and line are logged automatically by the logger since some time. So every log containing something  like this could be changed...